test/WebJobs.Script.Tests/Rpc/RpcWorkerConfigTests.cs

```diff
@@ -237,6 +237,32 @@ public void ValidateWorkerDescription_Succeeds(WorkerDescription workerDescripti
             }
         }
 
+        [Fact]
+        public void ValidateWorkerDescription_ResolvesDotNetDefaultWorkerExecutablePath()
+        {
+            var expectedExecutablePath =
+                RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
+                    ? Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ProgramFiles), "dotnet", "dotnet")
+                    : "dotnet";
+
+            var workerDescription = new RpcWorkerDescription { Language = testLanguage, Extensions = new List<string>(), DefaultExecutablePath = "dotnet" };
+            workerDescription.ApplyDefaultsAndValidate();
+            Assert.Equal(expectedExecutablePath, workerDescription.DefaultExecutablePath);
+        }
+
+        [Theory]
+        [InlineData("DotNet")]
+        [InlineData("dotnet.exe")]
```

Comment:
we should allow dotnet.exe as running dotnet.exe is same as running dotnet on windows